### PR TITLE
Remove PRODUCT_RELEASE, after that, all request need auth*.

### PR DESCRIPTION
**Pull request by zongzw**

### CODE REVIEW

File: app/waf/src/sequence.ts [hunk 1]
```diff
@@ -83,10 +83,13 @@ export class MySequence implements SequenceHandler {
   }
 
   async authRequest(context: RequestContext) {
-    if (!process.env.PRODUCT_RELEASE) return;
-
     let {request} = context;
 
+    let ignoredAuthPaths = ['/openapi.json', '/explorer'];
+    for (let u of ignoredAuthPaths) {
+      if (request.path.startsWith(u)) return;
```
Comment (zhang-shengping) [inline app/waf/src/sequence.ts]: If client request this path: /openapi.json', '/explorer', client will get nothing?

Comment (zongzw) [reply app/waf/src/sequence.ts]: In that case, the client can get a response without auth.

Comment (zhaoqin-github) [reply app/waf/src/sequence.ts]: Should we allow 'ping' without auth token? Or, which api endpoint should we use for healthcheck?

Comment (zongzw) [reply app/waf/src/sequence.ts]: The `ping` calls as3 API. I think that we should not allow anyone to `ping` without auth*. 
The client who wants to ping should have the auth info which is used for later works.

Comment (zhaoqin-github) [reply app/waf/src/sequence.ts]: A healthcheck call may not need to include openstack user token. That is not the topic of this pr, let us do that later.

Comment (zongzw) [reply app/waf/src/sequence.ts]: yea, we can add it when necessary.

**Comment by zongzw** [comment]
hold on, not to merge. let PR #170 be merged in first.

**Pull request merged**